LICENSE

```diff
@@ -223,7 +224,7 @@
    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
    SOFTWARE.
 
-   BSD-3 LICENSE
+   BSD-2 LICENSE
```

Comment:
Thanks Bhargavi: you need to delete the third point as well. ("3. Neither the name..." to change the license from BSD-3 to BSD-2.

Reply:
ok sure